versions/3.0.3.md

```diff
@@ -102,8 +102,6 @@ The OpenAPI Specification is versioned using [Semantic Versioning 2.0.0](https:/
 
 The `major`.`minor` portion of the semver (for example `3.0`) SHALL designate the OAS feature set. Typically, *`.patch`* versions address errors in this document, not the feature set. Tooling which supports OAS 3.0 SHOULD be compatible with all OAS 3.0.\* versions. The patch version SHOULD NOT be considered by tooling, making no distinction between `3.0.0` and `3.0.1` for example.
 
-Subsequent minor version releases of the OpenAPI Specification (incrementing the `minor` version number) SHOULD NOT interfere with tooling developed to a lower minor version and same major version.  Thus a hypothetical `3.1.0` specification SHOULD be usable with tooling designed for `3.0.0`.
```

Comment:
If an OAS definition has its minor version incremented, it MUST still be a valid definition for the new minor version.

E.g., if we have a document that is `3.0.2` and we change the version to `3.1.0`, then it MUST be a valid 3.1.0 document.

Reply:
(updated @darrelmiller 's text with "minor" changes)

Reply:
Done